### ISSUE TITLE
Problem when running "pertrained" AlexNet from Digits in imagenet.

### ISSUE THREAD

**Issue by schglmnn** [dusty-nv/jetson-inference#58]
Hello,
I have trained a pretrained AlexNet on my own dataset in digits. When running it on my TX1 with a modified "imagenet-console", I get an error. (When running an AlexNet that was not pretrained, everything works fine)

----------------------------------------------------------------------------------------------------
[GIE]  attempting to open cache file corridor/snapshot_iter.caffemodel.tensorcache
[GIE]  cache file not found, profiling network model
[GIE]  platform has FP16 support.
[GIE]  loading corridor/deploy.prototxt corridor/snapshot_iter.caffemodel
[libprotobuf ERROR google/protobuf/text_format.cc:290] Error parsing text-format ditcaffe.NetParameter: 42:18: Message type "ditcaffe.BatchNormParameter" has no field named "scale_filler".
Could not parse deploy file
[GIE]  failed to parse caffe network
failed to load corridor/snapshot_iter.caffemodel
failed to load corridor/snapshot_iter.caffemodel
imageNet -- failed to initialize.
imagenet-console:   failed to initialize imageNet
----------------------------------------------------------------------------------------------------

**Comment by MichaelX99** [comment]
I have run into the same problem except with googlenet, have you found a solution to this yet by any chance?
